recipes-tag/08-id-iss-analysis/meta.yaml

```diff
@@ -16,4 +16,5 @@ requirements:
     - netcdf4
     - numexpr
     - periodictable
-    - xraylarch
+    - pyzbar
+    - xraylarch >=0.9.45
```

Comment:
This is to use the updated `xraylarch` package from https://github.com/NSLS-II/lightsource2-recipes/pull/743.